src/spyglass/utils/dj_helper_fn.py

```diff
@@ -465,3 +466,41 @@ def make_file_obj_id_unique(nwb_path: str):
         f.attrs["object_id"] = new_id
     _resolve_external_table(nwb_path, nwb_path.split("/")[-1])
     return new_id
+
+
+def populate_pass_function(value):
```

Comment:
This seems like it could be a method of the mixin to live adjacent to the populate definition where it is used

Reply:
Having it as a method actually leads to the `SSLContext` errors as well.  Since it doesn't depend on the table instance it's easier to have it as a seperate function